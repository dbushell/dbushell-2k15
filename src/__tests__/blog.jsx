import React from 'react';
import Blog from '../components/blog';
import {outerHTML} from './utils';

describe('Blog component', () => {
  it(`renders with defaults`, () => {
    expect(
      outerHTML(
        <Blog/>
      )
    ).toMatchSnapshot();
  });
});
