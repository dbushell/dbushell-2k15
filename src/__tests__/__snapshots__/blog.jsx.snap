// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Blog component renders with defaults 1`] = `
"<aside class=\\"b-blog\\" role=\\"navigation\\">
  <div class=\\"b-blog__title\\">
    <h3>From the blogâ€¦</h3>
  </div>
  <ul class=\\"b-blog__list\\">
    <li class=\\"b-blog__item\\"><a rel=\\"bookmark\\" href=\\"/2020/03/27/debugging-a-todo-app/\\">Debugging a Todo App</a><time class=\\"time\\" dateTime=\\"2020-03-27T10:00:00.000Z\\">Friday <b>27 <abbr title=\\"March\\">Mar</abbr> 2020</b></time></li>
    <li class=\\"b-blog__item\\"><a rel=\\"bookmark\\" href=\\"/2020/03/05/bundle-a-pwa-as-an-android-app/\\">Bundle a PWA as an Android App</a><time class=\\"time\\" dateTime=\\"2020-03-05T10:00:00.000Z\\">Thursday <b>5 <abbr title=\\"March\\">Mar</abbr> 2020</b></time></li>
    <li class=\\"b-blog__item\\"><a rel=\\"bookmark\\" href=\\"/2020/02/28/http-security-headers-and-inline-resources/\\">HTTP Security Headers and Inline Resources</a><time class=\\"time\\" dateTime=\\"2020-02-28T10:00:00.000Z\\">Friday <b>28 <abbr title=\\"February\\">Feb</abbr> 2020</b></time></li>
    <li class=\\"b-blog__item\\"><a rel=\\"bookmark\\" href=\\"/2020/02/07/docker-wordpress-portless-localhost-domains/\\">Docker, WordPress, and Portless Localhost Domains</a><time class=\\"time\\" dateTime=\\"2020-02-07T10:00:00.000Z\\">Friday <b>7 <abbr title=\\"February\\">Feb</abbr> 2020</b></time></li>
    <li class=\\"b-blog__item\\"><a rel=\\"bookmark\\" href=\\"/2020/01/27/building-a-pwa-with-netlify-functions/\\">Building a PWA with Netlify Functions</a><time class=\\"time\\" dateTime=\\"2020-01-27T10:00:00.000Z\\">Monday <b>27 <abbr title=\\"January\\">Jan</abbr> 2020</b></time></li>
    <li class=\\"b-blog__item\\"><a rel=\\"bookmark\\" href=\\"/2020/01/23/a-morning-with-vuejs/\\">A Morning with Vue.js</a><time class=\\"time\\" dateTime=\\"2020-01-23T10:00:00.000Z\\">Thursday <b>23 <abbr title=\\"January\\">Jan</abbr> 2020</b></time></li>
  </ul>
</aside>"
`;
